org.openjkdf2.OpenJKDF2.yml

```diff
@@ -0,0 +1,32 @@
+app-id: org.openjkdf2.OpenJKDF2
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+
+command: openjkdf2-64
+
+modules:
+  - packaging/flatpak/glew/glew.json
+  - packaging/flatpak/glu/glu-9.json
+  - packaging/flatpak/python3-cogapp.json
+  
+  - name: openjkdf2-64
+    buildsystem: simple
+    build-commands:
+      - bash packaging/flatpak/build_linux64_flatpak.sh
```

Comment:
by Linux 64, you mean any 64-bits architecture, right?

Reply:
Yes, I test primarily on AArch64, but I also support x86_64. I guess 32-bit is technically supported though (I just don't test on it often) so I'll rename this.

Reply:
we only support these two 64-bits arch.

I'm surprised that for once "linux64" doesn't equate x86_64. :-)